<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Surendar-GSR" ignored="0" name="nopCommerce Test1" tests="5" failures="3" timestamp="30 Dec 2021 07:08:57 GMT" time="150.068" errors="0">
  <testcase name="LoginTest" time="36.791" classname="com.nopEcommerce.test_cases.TestCase_LoginTest_001"/>
  <testcase name="logintest" time="26.938" classname="com.nopEcommerce.test_cases.TestCase_LoginDDT_002"/>
  <testcase name="logintest" time="11.427" classname="com.nopEcommerce.test_cases.TestCase_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.nopEcommerce.test_cases.TestCase_LoginDDT_002.logintest(TestCase_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase name="logintest" time="11.7" classname="com.nopEcommerce.test_cases.TestCase_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.nopEcommerce.test_cases.TestCase_LoginDDT_002.logintest(TestCase_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase name="logintest" time="10.182" classname="com.nopEcommerce.test_cases.TestCase_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.nopEcommerce.test_cases.TestCase_LoginDDT_002.logintest(TestCase_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- nopCommerce Test1 -->
